apps/checkout-app/frontend/misc/errors.ts

```diff
@@ -0,0 +1,23 @@
+import { MetadataErrorCode } from "@/graphql";
+import { IntlShape } from "react-intl";
+import { commonErrorMessages } from "./errorMessages";
+
+export const getMetadataErrorMessage = (
+  code: MetadataErrorCode,
+  intl: IntlShape
+) => {
+  switch (code) {
+    case "INVALID":
```

Comment:
I think since these message names do match with code names this could be shortened to
```suggestion
    const errorMessage = commonErrorsMessages[camelCase(code)] || commonErrorMessages.unknown;
    
    return intl.formatMessage(errorMessage);
```

`camelCase` being a lodash fn